src/stores/results.ts

```diff
@@ -10,34 +9,59 @@ export const useResultsStore = defineStore("results", {
       return this.resultats?.droitsEligibles || []
     },
     benefitTreeGroupExperiment(): (StandardBenefit | BenefitGroup)[] {
-      const bafaGroup: BenefitGroup = {
-        benefits: [],
-        id: "bafa-bafd-group",
-        label: "Aides BAFA et BAFD",
-        logoPath: "/img/benefits/logo-bafa-bafd.png",
-        description:
-          "Différents organismes peuvent vous aider à financer votre formation BAFA ou BAFD.",
-        redirectionPage: "bafa-bafd",
-      }
+      const groups: BenefitGroup[] = [
+        {
+          benefits: [],
+          id: "bafa-bafd-group",
+          label: "Aides BAFA et BAFD",
+          logoPath: "/img/benefits/logo-bafa-bafd.png",
+          description:
+            "Différents organismes peuvent vous aider à financer votre formation BAFA ou BAFD.",
+          interestFlag: "_interetBafa",
+        },
+        {
+          benefits: [],
+          id: "velo-group",
+          label: "Aides à l'achat d'un vélo",
+          logoPath: "/img/benefits/logo-velo.jpg",
+          description:
+            "Différents organismes peuvent vous aider à financer votre vélo.",
+          interestFlag: "_interetsAidesVelo",
+        },
+      ]
+
+      const benefitGroups = groups.filter(
+        (benefitGroup) =>
+          this.benefits.filter(
+            (benefit) => benefit.interestFlag === benefitGroup.interestFlag
+          ).length > 1
+      )
 
       const results = this.benefits.reduce((results, benefit) => {
-        if (hasBafaInterestFlag(benefit)) {
-          if (bafaGroup.benefits.length === 0) {
-            results.push(bafaGroup)
+        let inGroup = false
+        benefitGroups.forEach((benefitToGroup) => {
+          if (
+            !inGroup &&
+            benefit?.interestFlag === benefitToGroup.interestFlag
+          ) {
+            if (benefitToGroup.benefits.length === 0) {
+              results.push(benefitToGroup)
+            }
+
+            benefitToGroup.benefits.push(benefit)
+            benefit.groupLabel = benefitToGroup.label
+            inGroup = true
           }
-          bafaGroup.benefits.push(benefit)
-          benefit.groupLabel = bafaGroup.label
-        } else {
+        })
```

Comment:
Remarque pas bloquante; ici je pense que conserver une méthode/condition pour tester directement si l'aide a un `interestFlag` à l'entrée serait plus performante car on va tester les deux groupes pour chaque aide

Reply:
Tout à fait d'accord avec toi. Mais je pense à itérer pour un prochain ticket en enlevant l'AB testing et en intégrant plus simplement le groupement dans les fiches peut-être ?

Reply:
https://trello.com/c/WsOQLPwh/1678-update-de-performance-groupement-des-aides